Sources/App/Controllers/MemoController.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  File.swift
+//  
+//
+//  Created by 천수현 on 2021/07/06.
+//
+
+import Fluent
+import Vapor
+
+enum SortOrder: String, Decodable {
+    case ascending = "ascending"
+    case descending = "descending"
```

Comment:
case의 이름과 rawValue가 같은 경우 지정하지 않아도 된답니다.

```suggestion
    case ascending
    case descending
```